CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,7 @@
+# Community Code of Conduct
+
+Cluster.dev follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+<!-- markdownlint-disable no-inline-html -->
+Instances of abusive, harassing, or otherwise unacceptable behavior in Cluster.dev may be reported by contacting the SHALB via <SOME_EMAIL@shalb.com>
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behaviour in Cluster.dev should be reported by contacting SHALB via <SOME_EMAIL@shalb.com>
```